ntbs-integration-tests/NotificationPages/MBovisAnimalExposurePageTests.cs

```diff
@@ -68,12 +83,12 @@ public async Task IfNotificationHasKnownCases_DisplaysTable()
             Assert.NotNull(document.QuerySelector("#mbovis-animal-exposure-table"));
         }
 
-        [Fact]
-        public async Task IfNotificationDoesNotHaveKnownCases_DoesNotDisplayTable()
+        [Theory]
+        [InlineData(Utilities.NOTIFICATION_ID_WITH_MBOVIS_NULL_ANIMAL_NO_ENTITIES)]
+        [InlineData(Utilities.NOTIFICATION_ID_WITH_MBOVIS_NO_ANIMAL_NO_ENTITIES)]
+        [InlineData(Utilities.NOTIFICATION_ID_WITH_MBOVIS_UNKNOWN_ANIMAL_NO_ENTITIES)]
+        public async Task IfNotificationDoesNotHaveKnownCases_DoesNotDisplayTable(int notificationId)
```

Comment:
I won't ask you to change this, but I would question the value of having a separate test cases for each of these. All of these three tests essentially say the same thing: "When a given property on the model is `false`, a table is not displayed on the page". Testing the different ways that this property can be `false` would be better suited to unit tests, because these are much quicker to run.

Writing automated tests can often be a trade-off between how much functionality you're testing and how long the tests take to run.

Reply:
Thanks for that, I'll bear it in mind next time!